.github/workflows/notification-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Notification: PR'
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  notification:
+    name: Notification
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send telegram message on pr events
+        uses: appleboy/telegram-action@master
+        with:
+          to: ${{ secrets.TELEGRAM_RECIPIENT }}
```

Comment:
I am specified as the recipient, you will not receive these notifications. As far as I understand, the action does not allow you to send notifications to all subscribers of the bot.

Reply:
I've replaced telegram sender for more opportunities